worldify/scripts/spotify_lib.py

```diff
@@ -1,11 +1,13 @@
 '''Containing functions to interact with spotify web api wrapper'''
 
+import datetime
+import os
+import requests
+import urlparse
+
 from spotipy.oauth2 import SpotifyClientCredentials
 import spotipy
-import os
 import spotipy.util as util
-import requests
-import datetime
 
 # Constants
 SPOTIFY_USER_ID = os.getenv("SPOTIFY_USER_ID")
```

Comment:
With the merge of pull #4 we now use a config parser instead you can access a global instance of this within the constants file [0]. The layout of the file can be found here [1]

[0] https://github.com/Worldify/Worldify/blob/master/worldify/constants.py#L3
[1] https://github.com/Worldify/Worldify#config-requirements